netaddr_test.go

```diff
@@ -738,10 +738,6 @@ func TestParseIPPrefixError(t *testing.T) {
 			prefix: "192.168.0.0",
 			errstr: "no '/'",
 		},
-		{
```

Comment:
please don't delete test cases. You can update them to show the new behavior, though.

Reply:
fixed the err string, but this seems to be testing lower than what ParseIPPrefix should be concerned with right? This test should (and is) be a case for ParseIP.